configs/webpack.config.renderer.dev.babel.js

```diff
@@ -118,10 +118,13 @@ export default merge(baseConfig, {
         test: /^((?!\.global).)*\.(scss|sass)$/,
         use: [
           {
-            loader: 'typings-for-css-modules-loader',
+            loader: 'style-loader',
+          },
+          {
+            loader: '@teamsupercell/typings-for-css-modules-loader',
```

Comment:
I don't know the context of the fix. Can you provide some context on how this fixes the bug

Reply:
I think there are 2 parts to this fix
- Re-added node-sass which was accidentally removed in v1.2.0
- Replaced `typings-for-css-modules-loader` with an actively maintained fork
  - [See thread](https://github.com/Jimdo/typings-for-css-modules-loader/issues/94)

Reply:
I see, makes sense. Good call!